x-pack/dev-tools/functional_tests/lib/index.js

```diff
@@ -6,7 +6,6 @@
 
 export { getFtrConfig } from './get_ftr_config';
 
-export { runKibanaServer } from './run_kibana_server';
 export { runEsWithXpack } from './run_es_with_xpack';
```

Comment:
Since running with or without x-pack is an option presented to `kbn/es` with `license`. We can probably get rid of this altogether and add as part of the configuration.